Test failures with GEOS master

We currently have some failing tests in the GEOS master CI build (which is allowed to fail, so it's not "red"):

```
=================================== FAILURES ===================================
______________________ OperationsTestCase.test_operations ______________________

self = <tests.test_operations.OperationsTestCase testMethod=test_operations>

    def test_operations(self):
        point = Point(0.0, 0.0)
    
        # General geometry
        self.assertEqual(point.area, 0.0)
        self.assertEqual(point.length, 0.0)
        self.assertAlmostEqual(point.distance(Point(-1.0, -1.0)),
                               1.4142135623730951)
    
        # Topology operations
    
        # Envelope
        self.assertIsInstance(point.envelope, Point)
    
        # Intersection
        self.assertTrue(point.intersection(Point(-1, -1)).is_empty)
    
        # Buffer
        self.assertIsInstance(point.buffer(10.0), Polygon)
        self.assertIsInstance(point.buffer(10.0, 32), Polygon)
    
        # Simplify
        p = loads('POLYGON ((120 120, 121 121, 122 122, 220 120, 180 199, '
                  '160 200, 140 199, 120 120))')
        expected = loads('POLYGON ((120 120, 140 199, 160 200, 180 199, '
                         '220 120, 120 120))')
        s = p.simplify(10.0, preserve_topology=False)
>       self.assertTrue(s.equals_exact(expected, 0.001))
E       AssertionError: False is not true

tests/test_operations.py:37: AssertionError
---------------------------- Captured stdout setup -----------------------------
Speedups enabled for test_operations.
___________________________________ test_wkt ___________________________________

pipi4 = <shapely.geometry.point.Point object at 0x7ff9279b9970>

    def test_wkt(pipi4):
        """.wkt and wkt.dumps() both do not trim by default."""
>       assert pipi4.wkt == "POINT ({0:.14f} {1:.14f})".format(pi*4, -pi*4)
E       AssertionError: assert 'POINT (12.56...370614359172)' == 'POINT (12.56...637061435917)'
E         - POINT (12.56637061435917 -12.56637061435917)
E         + POINT (12.566370614359172 -12.566370614359172)
E         ?                         +                   +

tests/test_wkt.py:26: AssertionError
```

I assume both failures are relatively innocent, and the tests just need to be updated to pass (the first one is probably a normalization issue, and for the second one there have been indeed some changes in GEOS recently regarding WKT precision).

--- a/tests/test_operations.py
+++ b/tests/test_operations.py
@@ -29,8 +29,8 @@ class OperationsTestCase(unittest.TestCase):
         self.assertIsInstance(point.buffer(10.0, 32), Polygon)
 
         # Simplify
-        p = loads('POLYGON ((120 120, 121 121, 122 122, 220 120, 180 199, '
-                  '160 200, 140 199, 120 120))')
+        p = loads('POLYGON ((120 120, 140 199, 160 200, 180 199, 220 120, '
+                  '122 122, 121 121, 120 120))')
         expected = loads('POLYGON ((120 120, 140 199, 160 200, 180 199, '
                          '220 120, 120 120))')
         s = p.simplify(10.0, preserve_topology=False)

